massa-client/src/cmds.rs

```diff
@@ -1113,7 +1113,7 @@ impl Command {
                 }
             }
             Command::read_only_execute_smart_contract => {
-                if parameters.len() != 2 && parameters.len() <= 4 {
+                if parameters.len() < 2 || parameters.len() > 5 {
```

Comment:
Unless I'm tired, it should be
```rust
if parameters.len() < 2 || parameters.len() > 4
```
right?
If you have 5 params it should bail ( read_only_execute_smart_contract PathToBytecode MaxGas Address IsFinal )

Reply:
Copy and paste is evil when you don't have automated test. I'll recheck for the 100 time

Reply:
done

Reply:
Nice! Improving the client, one PR at a time 👍